h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -63,7 +63,7 @@
  */
 public class EasyPredictModelWrapper implements Serializable {
   // These private members are read-only after the constructor.
-  public final GenModel m;
```

Comment:
I think it should stay `final` - any reason to change?

Reply:
🤦 no..